back/src/test/java/com/cocktailpick/back/user/docs/UserDocumentation.java

```diff
@@ -25,4 +25,14 @@ public static RestDocumentationResultHandler findMe() {
 			)
 		);
 	}
+
+	public static RestDocumentationResultHandler updateUser() {
+		return document("user/me",
+			requestHeaders(
+				headerWithName("authorization").description("Bearer 토큰")),
+			responseFields(
+				fieldWithPath("name").type(JsonFieldType.STRING).description("접속한 user의 이름")
```

Comment:
```suggestion
				fieldWithPath("name").type(JsonFieldType.STRING).description("변경할 user의 이름")
```
요청으로 '바뀔 이름'이 오는거니까 `변경할 user의 이름`같은 느낌으로 바꾸는 것이 어떨까요? 아니면 `요청한 user의 이름`?? 이름 짓기가 어렵네요.. 😢

Reply:
변경할 수정할 바뀔
골라골라~
🙂

Reply:
`수정할` 이 가장 와닿는 느낌이 드네요!